Gems/GraphModel/Code/Source/Model/Slot.cpp

```diff
@@ -495,6 +508,11 @@ namespace GraphModel
         return m_slotDefinition->GetDescription();
     }
 
+    const AZStd::vector<AZStd::string>& Slot::GetEnumValues() const
+    {
+        return m_slotDefinition->GetEnumValues();
+    }
+
```

Comment:
Nice!

Reply:
Thanks! I will also follow up and expose the enum values so they can show up on slots. The graph controller is setting up most of the supported types except for those that require additional data like this one.